src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,16 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        if ((result1 == bet1) && (result2 == bet2)) {
```

Comment:
good start) 
but read [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html) attentively
- Use ternary operator instead of if-else

Reply:
Agree. I have changed the code.